### DISCUSSION TITLE
why .classpath files don't appear in file explorer?

### DISCUSSION THREAD

**Discussion by Ignacio-Ucedo** [zed-industries/zed#17593] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

mkdir test
touch test/.classpath
zeditor test

at least in my environment, the .classpath file is not visible

### Environment

Zed: v0.151.2 (Zed)
OS: Linux X11 arch unknown
Memory: 14.9 GiB
Architecture: x86_64
GPU: AMD Radeon Graphics (RADV RENOIR) || radv || Mesa 24.2.2-arch1.1

**Comment by SomeoneToIgnore** [comment]
https://github.com/zed-industries/zed/blob/41a61a978a62b66d63fbebf913f48fe486f09cbd/assets/settings/default.json#L549-L561

That's why.
Consider copying this part into your default settings and removing the line with classpath if you need them.

**Reply by Ignacio-Ucedo** [reply]
Great, thanks!
